Gemfile.lock

```diff
@@ -49,7 +49,7 @@ GEM
     minitest (5.13.0)
     money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.7)
+    nokogiri (1.10.8)
```

Comment:
### [nokogiri](https://nokogiri.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.10.7 → 1.10.8

[_[change-log](https://nokogiri.org/CHANGELOG.html), [source-code](https://github.com/sparklemotion/nokogiri)_]

:tada: Patched vulnerabilities:

- [CVE-2020-7595](https://nvd.nist.gov/vuln/detail/CVE-2020-7595)
  **libxml2 2.9.10 has an infinite loop in a certain end-of-file situation**

  CVSS V2: [5.0 medium](https://nvd.nist.gov/vuln-metrics/cvss/v2-calculator?name=CVE-2020-7595)
  URL: https://github.com/sparklemotion/nokogiri/issues/1992

  Nokogiri has backported the patch for CVE-2020-7595 into its vendored version of libxml2, and released this as v1.10.8  CVE-2020-7595 has not yet been addressed in an upstream libxml2 release, and so Nokogiri versions <= v1.10.7 are vulnerable.






<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/sparklemotion/nokogiri/compare/v1.10.7...v1.10.8).

These are the individual commits:
- (2cdb68e) [backport libxml2 patch for CVE-2020-7595](https://github.com/sparklemotion/nokogiri/commit/2cdb68e95aa075ac36a08d4d82d9b410a950a051)
- (570b6cb) [update to use rake-compiler ~1.1.0](https://github.com/sparklemotion/nokogiri/commit/570b6cbc5fbc5ee7ef969332c587b951ae35bcd0)
- (4a77fdb) [remove patches from the hoe Manifest](https://github.com/sparklemotion/nokogiri/commit/4a77fdb789aefed7ca65c7c7f57ad4dca0d3b209)
- (2320f5b) [update CHANGELOG for v1.10.8](https://github.com/sparklemotion/nokogiri/commit/2320f5bd6319dca9c68d85bbf41629bbf8052a49)
- (6ce10d1) [version bump to v1.10.8](https://github.com/sparklemotion/nokogiri/commit/6ce10d15d7af6ad65813a495eaf168f73eba211c)


</details>